### PR TITLE
Bicep DLL v0.4.1008 support

### PR DESCRIPTION
**Pull request by PalmEmanuel**
Also changed the path of the Bicep source to "..\..\bicep\" instead of "..\TMP\bicep\". This allows you to clone both repositories into the same directory and then just build BicepNet.

### CODE REVIEW

**Comment by StefanIvemo** [comment]
Done a bunch of tests using the module with PSBicep and it looks good overall. I do have one issue though where it ignores my `bicepconfig.json` files. See image below where I have a `bicepconfig.json` file that sets the linter rule `no-unused-params` to `off`. Bicep CLI uses the config file `Build-Bicep` does not use it.

image

File: BicepNet.Core/BicepNet.Core.csproj [hunk 1]
```diff
@@ -6,8 +6,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\TMP\bicep\src\Bicep.Core\Bicep.Core.csproj" />
-    <ProjectReference Include="..\TMP\bicep\src\Bicep.Decompiler\Bicep.Decompiler.csproj" />
+    <ProjectReference Include="..\..\bicep\src\Bicep.Core\Bicep.Core.csproj" />
```
Comment (StefanIvemo) [inline BicepNet.Core/BicepNet.Core.csproj:9]: `.\scripts\Dependencies.ps1` still downloads files to `..\TMP`

**Comment by PalmEmanuel** [comment]
> Done a bunch of tests using the module with PSBicep and it looks good overall. I do have one issue though where it ignores my `bicepconfig.json` files. See image below where I have a `bicepconfig.json` file that sets the linter rule `no-unused-params` to `off`. Bicep CLI uses the config file `Build-Bicep` does not use it.
> 
> image

Did we build support for the bicepconfig.json yet? I thought that wasn't implemented but only a planned feature/in review, as noted in https://github.com/PSBicep/PSBicep/issues/170.

**Comment by StefanIvemo** [comment]
> Did we build support for the bicepconfig.json yet? I thought that wasn't implemented but only a planned feature/in review, as noted in [PSBicep/PSBicep#170](https://github.com/PSBicep/PSBicep/issues/170).

The issue referenced is for a new command that helps the user create a new `bicepconfig.json` file. Support for `bicepconfig.json` is already implemented in the module.

**Comment by PalmEmanuel** [comment]
> > Did we build support for the bicepconfig.json yet? I thought that wasn't implemented but only a planned feature/in review, as noted in [PSBicep/PSBicep#170](https://github.com/PSBicep/PSBicep/issues/170).
> 
> The issue referenced is for a new command that helps the user create a new `bicepconfig.json` file. Support for `bicepconfig.json` is already implemented in the module.

I see, then this change probably broke it because I added a new bicepconfig file (or maybe it was in #5?) to allow for the publish and restore url configuration. I'll see if I can change that.

**Comment by StefanIvemo** [comment]
Call me if we should take a look at it together 😊.

**Comment by PalmEmanuel** [comment]
This is replaced by #5 which also implements the new functionality.

**Pull request closed without merging**